doc/source/whatsnew/v1.2.4.rst

```diff
@@ -35,7 +35,7 @@ Bug fixes
 Other
 ~~~~~
 
--
+- Allow :class:`Index` to be passed to the numpy ``all`` function.
```

Comment:
move to 1.3 Bug Fixes in reshaping section, add the PR number as the issue number, can you reference np.all here I think `:func:numpy.all` works or similar